eahub/templates/eahub/profile.html

```diff
@@ -39,81 +39,102 @@
+        </div>
+      {% endif %}
+
+      <div class="col-xs-24 col-lg-18">
         <div class="prof__title row">
-          <h1 class="col col-auto">{{ profile.name }}</h1>
+          <h1 class="col col-auto">{{ profile.get_full_name }}</h1>
 
           <div class="col col-auto d-flex">
             {% if profile.facebook_url %}
               <a class="prof_social-link" href="{{ profile.facebook_url }}"
                 rel="nofollow"
               >
-                <i class="fab fa-facebook"></i>
+                <i class="fab fa-facebook-square"></i>
               </a>
             {% endif %}
-            {% if profile.personal_website_url %}
-              <a class="prof_social-link"
-                href="{{ profile.personal_website_url }}" rel="nofollow"
+          
+            {% if profile.linkedin_url %}
+              <a class="prof_social-link" href="{{ profile.linkedin_url }}"
+                rel="nofollow"
               >
-                <i class="fa fa-link"></i>
+                <i class="fab fa-linkedin"></i>
               </a>
             {% endif %}
-            {% if profile.linkedin_url %}
-              <a class="prof_social-link" href="{{ profile.linkedin_url }}"
+          
+            {% if profile.personal_website_url %}
+              <a class="prof_social-link"
+                href="{{ profile.personal_website_url }}"
                 rel="nofollow"
+                data-bs-content="Personal website" data-bs-toggle="popover" data-bs-container="body" data-bs-trigger="hover focus" data-bs-placement="bottom"
               >
-                <i class="fab fa-linkedin-in"></i>
+                <i class="fa fa-link"></i>
               </a>
             {% endif %}
           </div>
 
+          {% if profile.tags_generic.exists %}
+            <div class="prof__tags col col-24 col-lg-auto align-self-start">
+              {% for tag in profile.get_tags_generic_formatted %}
+                <span class="prof__tag">{{ tag }}</span>
+              {% endfor %}
+            </div>
+          {% endif %}
+        
+          {% if profile.job_title %}
+            <div class="prof__job-title col col-24">{{ profile.job_title }}</div>
+          {% endif %}
+
           {% if request.user.id == profile.user_id %}
-            <div class="prof__btns col col-24">
-              <a class="btn btn-outline-primary btn-sm"
-                href="{% url 'account_email' %}"
-              >
-                <span><i class="fa fa-pencil-alt"
-                ></i> Change Email Address</span>
-              </a>
-              <a class="btn btn-outline-primary btn-sm"
-                href="{% url 'account_change_password' %}"
-              >
-                <span><i class="fa fa-pencil-alt"></i> Change Password</span>
-              </a>
-              <a class="btn btn-outline-primary btn-sm"
-                href="{% url 'edit_profile' %}"
+            <div class="prof__btns col col-24 d-none d-lg-flex">
```

Comment:
(optional) I think Edit button and social media buttons should be at same height, looks a bit wonky.
If you don't want to do this now, you could add this to some list for to be done later

![image](https://user-images.githubusercontent.com/15908208/112803642-73955900-906b-11eb-8064-cf02fac6f8cc.png)

Reply:
yes, it's a firefox bug

FYI I develop only on chrome, hence until multi-browser testing is done I don't look at other browser